src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj

```diff
@@ -3,9 +3,10 @@
   <PropertyGroup>
     <Description>Microsoft Azure Monitor Library</Description>
     <AssemblyName>Microsoft.Azure.Management.Monitor</AssemblyName>
-    <Version>0.16.1-preview</Version>
+    <Version>0.17.0-preview</Version>
     <PackageId>Microsoft.Azure.Management.Monitor</PackageId>
     <PackageTags>Management.Monitor;Management.Monitoring;metrics;alerts;autoscale;activityLogs;events;operations;logs</PackageTags>
+    <PackageReleaseNotes>See https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/src/SDKs/Monitor/changelog.md for release notes.</PackageReleaseNotes>
```

Comment:
This URL will be of no use once psSdkJson6 branch is deleted
Rather create an aka.ms link for your changelog.md
Will allow it for now, but consider the above option for future releases

Reply:
Indeed, thanks for catching that. Will update to:
https://aka.ms/azure-sdk-monitor